common/src/main/java/com/wynntils/core/config/ConfigManager.java

```diff
@@ -169,13 +152,22 @@ public void loadConfigOptions(boolean resetIfNotFound, boolean initOverlayGroups
         }
 
         // Newly created group overlays need to be enabled
-        for (OverlayGroupHolder holder : OVERLAY_GROUP_FIELDS) {
-            holder.getParent().enableOverlays();
+        for (OverlayGroupHolder holder : Managers.Overlay.getOverlayGroups()) {
```

Comment:
Is it really the responsibility of ConfigManager to enable overlays? Or does `enableOverlay` just do config-related stuff and has a misleading name?

Reply:
This does not enable overlays, rather reads the config file and creates the neccessary amount of instances. This is something I would like to fix in the follow-up, but this needs a lot of thinking.

Reply:
(overall this still improved from what it used to be)

Reply:
Well, the `enableOverlays` method seems it does thing that I should describe as "enable the overlays", but I accept that we might have to return here a second time around to fix this better.